contracts/AccountTree.sol

```diff
@@ -78,21 +78,18 @@ contract AccountTree {
             "AccountTree: right set is full "
         );
 
-        bytes32[BATCH_SIZE / 2] memory buf;
```

Comment:
unlike the merklise libaray case, which uses a dynamic array. Here the buf, pNodes, and leafs must match each other's array size.